skfda/_utils/_utils.py

```diff
@@ -447,3 +447,20 @@ def _check_estimator(estimator):
     instance = estimator()
     check_get_params_invariance(name, instance)
     check_set_params(name, instance)
+
+
+def _classifier_fit_init(y):
```

Comment:
Maybe the name could be more clear.

```suggestion
def _classifier_get_classes(y):
```